### ISSUE TITLE
Add support for RHEL/Rocky 8.9

### ISSUE THREAD

**Issue by coderforlife** [aws/aws-parallelcluster#5983]
**Required Info:**
 - AWS ParallelCluster version: 3.8.0

**Bug description and how to reproduce:**
When creating a custom AMI based on RHEL or Rocky 8.9 it fails to build since it cannot access the repo at from https://fsx-lustre-client-repo.s3.amazonaws.com/el/8.9/x86_64/repodata/repomd.xml (only 8.8 exists). It is entirely possible that using the same exact repo would in fact work with 8.9, but that is not easy to change (at least I haven't found an easy way, but I don't really understand how to update Chef cookbooks in this system).

Another option could be to allow images to be built without FSx/Lustre support at all and skip all steps for that.

**Comment by jdeamicis** [comment]
Hi, thanks for opening this feature request.

Currently the FSx-for-Lustre client does not support RHEL/Rocky 8.9 kernels (see the [FSx-for-Lustre client kernel compatibility matrix](https://docs.aws.amazon.com/fsx/latest/LustreGuide/lustre-client-matrix.html)).

We could consider a feature request to disable parts of the Chef cookbooks, but that's not something for the short term.

Alternatively, there's a way to use a custom cookbook to build an image. You can create a fork of the [cookbook project](https://github.com/aws/aws-parallelcluster-cookbook/) and disable the installation of the lustre client.

I *think* it should be enough to comment out [this line](https://github.com/aws/aws-parallelcluster-cookbook/blob/5daa911a8a6c61c6f0dadad97be45444ed81f9df/cookbooks/aws-parallelcluster-environment/recipes/install.rb#L22).

Would this approach satisfy you? If so, I can provide more information on how to use a custom cookbook at image build time.

**Comment by coderforlife** [comment]
That would be fantastic. There are a few other things I might want to change as well (such as disabling DCV which requires a massive amount of installs on Rocky because it has to install all of the Gnome desktop environment).

**Comment by jdeamicis** [comment]
If you fork the cookbook and you disable the Lustre installation as described above (to be tested),  you can use it in a development section of the image build configuration file:

```yaml
Region: ...
Build:
  ...
Image:
  ...
DevSettings:
  Cookbook:
    ChefCookbook: <path_to_github_repo+branch>.tar.gz
```

Please make sure you do your changes on top of the `release-3.8` branch on our repository.

Also, I'd say it's a good idea to use the same custom cookbook also at cluster creation, although in your very case it shouldn't matter. You can use the same `DevSettings` in your cluster configuration file.

**Comment by enrico-usai** [comment]
Related to this: https://github.com/aws/aws-parallelcluster/issues/5913

**Comment by enrico-usai** [comment]
ParallelCluster 3.9.0 has been released with RHEL/Rocky 8.9 support.
